docs/userguide/how_to_configure_simulations.rst

```diff
@@ -0,0 +1,331 @@
+How-to customize simulations using model transformations and evaluation modes
+=============================================================================
+
+Qiskit Dynamics provides model transformations and different numerical methods
+for evaluating models that can be used to customize how a simulation is done,
```

Comment:
Line 5 is a bit confusing, when I think it through it makes sense. But if you could split it up a bit that might be nice

Reply:
Agreed, just expanded this into what I think is a better paragraph.